capture_gui/lib.py

```diff
@@ -194,7 +195,7 @@ def _browse(path):
             default_filename = "playblast"
 
         # Default to images rule (if rule exists in workspace)
-        images_rule = mc.workspace("images", q=1, fre=True)
+        images_rule = pm.workspace.fileRules["images"]
```

Comment:
Ok, to get back to this: I believe the problem was coming from the fact that we were calling the code the wrong way.

We had:

```python
images_rule = mc.workspace("images", query=True, fileRuleEntry=True)
```

Where it should be:

```python
images_rule = mc.workspace(fileRuleEntry="images")
```

That means we should be able to remove the need for Pymel again with this change in there.